ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -116,6 +116,11 @@ data "openstack_networking_network_v2" "cluster_external_network" {
   name = "{{ cluster_external_network }}"
 }
 
+# Always get the SRIOV storage network 
+data "openstack_networking_network_v2" "portal_storage_direct" {
+  tags = ["portal-storage-direct"]
```

Comment:
Lets make this something like cluster_storage_network, and we skip attaching to it if its not defined via extra vars?